components/slider-carousel/src/ReuseSliderCarousel.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState, useRef, useEffect, CSSProperties } from "react";
+import { useSlider } from "@locoworks/reusejs-toolkit-react-hooks";
+import { twMerge } from "tailwind-merge";
+interface ReuseSliderCarouselInterface {
+  slideInterval: number;
```

Comment:
Does slideInterval have a default value. Since reuse styled components can have default properties, we can set a default interval(eg. 2000) here, making this optional.